src/Datadog.Trace.ClrProfiler.Managed/Integrations/AdoNet/NpgsqlCommandIntegration.cs

```diff
@@ -585,7 +585,7 @@ public static class NpgsqlCommandIntegration
         /// <param name="moduleVersionPtr">A pointer to the module version GUID.</param>
         /// <returns>The value returned by the instrumented method.</returns>
         [InterceptMethod(
-            TargetAssemblies = new[] { AdoNetConstants.AssemblyNames.SystemData, AdoNetConstants.AssemblyNames.SystemDataSqlClient },
+            TargetAssemblies = new[] { NpgsqlAssemblyName },
             TargetType = NpgsqlCommandTypeName,
             TargetSignatureTypes = new[] { "System.Threading.Tasks.Task`1<System.Object>", ClrNames.CancellationToken },
             TargetMinimumVersion = Major4,
```

Comment:
While you're here, can you revert the `Major5` change since `Npgqsl` isn't a built-in ADO.NET provider and shouldn't have been included in that update?

Reply:
Good catch. Fixed in f8f6f9b5 (the attribute here) and f2a3c573 (`integrations.json`).

Reply:
While I was there I also fixed the xml comments on every wrapper method. Hope you don't mind 😉